src/api/spec/controllers/webui/package_controller_binaries_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'webmock/rspec'
+require 'rails_helper'
+
+RSpec.describe Webui::PackageController, :vcr do
```

Comment:
@eduardoj hmm, but this is testing the `webui/package_controller.rb`, also the endpoints `#dependency` and `#wipe_binaries` are not part of the `webui/packages/binaries_controller.rb`.

Reply:
something looks wrong here, i think those test belong into the `webui/package_controller_spec.rb`

Reply:
@krauselukas, I just wanted to keep separated the file with the tests of `#dependency` and `#wipe_binaries`, and to rename it to something valid.

I also followed the naming convention described by the [RSpec/SpecFilePathFormat](https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/SpecFilePathFormat) cop. In case we had lots of methods to test in a class, instead of having only one test file (`my_class_spec.rb`) we could have several spec files (`my_class_file1_spec.rb`, `my_class_file2_spec.rb`, ...). That's the reason I decided to rename the `binaries.spec` file to `package_controller_binaries_spec.rb`.

Reply:
@eduardoj but since this is not testing endpoints defined in the `webui/packages/binaries_controller.rb` but the `webui/package_controller.rb` I would say its wrong. Like if I ever change those endpoints in the `package_controller.rb` I would not search for the corresponding tests in the `package_controller_binaries_spec.rb`. Also from what I understand from the cop it talking about the method name. `my_class_method_spec.rb  # describe MyClass, '#method'`

Reply:
I know, not introduced by you, but since you touching it, I would move those tests to the `package_controller_spec.rb` instead.

Reply:
@krauselukas I don't know why you wouldn't search for tests for the `Webui::PackageController` in files with the format `package_controller_*_spec.rb`. It solves the problem introduced when the tests were splitted into the `spec/controllers/webui/package_controller/binaries.spec` file. I don't see a less intrusive alternative.

I don't have any issue with leaving the tests for the `dependency` and `wipe_binaries` in the `package_controller_binaries_spec.rb` file.

From the last sentence you said about cop and the method name, it is still not clear to me if you are suggesting to split the tests into more files or not, like these ones:
- `package_controller_spec.rb`
- `package_controller_dependency_spec.rb`
- `package_controller_wipe_binaries_spec.rb`

I'm also fine with this approach.

It is clear you feel something is wrong with the current approach. Sorry for that. What option are you really suggesting?
1. to keep the tests in one file
2. to split the tests in three files, like above

Reply:
I'm fine with either moving the tests  `#wipe_binaries` and `#dependency` to `package_controller_spec.rb` or creating `package_controller_dependency_spec.rb` and `package_controller_wipe_binaries_spec.rb`.

Reply:
Just not as it is right now with `package_controller_binaries_spec.rb`, because there is no method/endpoint in `package_controller.rb` called `binaries`

Reply:
I created the `package_controller_dependency_spec.rb` and `package_controller_wipe_binaries_spec.rb` files and splitted the tests. Please take a look again.